src/spryker/docker/image/jenkins-runner/root/entrypoint.jenkins_runner.dynamic.sh

```diff
@@ -12,4 +12,4 @@ bootstrap()
 }
 
 bootstrap
-main
+main $@
```

Comment:
Is shellcheck happy with this?

Reply:
No, its not (never heard of it until you mentioned it) - I will update.